apps/frontend/src/app/pages/company/gpts/gpts.component.html

```diff
@@ -0,0 +1,347 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
+  <div class="pb-4 pt-6 flex flex-wrap justify-between gap-6">
+    <h2 class="font-sans font-medium text-4xl text-base-100">GPTs</h2>
+    @if(gptType === 'FINETUNING'){
+    <button
+      class="btn btn-primary btn-outline w-52"
+      onclick="create_fine_turning_modal.showModal()"
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/4f8f41e4-ad15-403f-be3b-10ad51931125)

When I click on this isn't the modal below supposed to appear? 

![image](https://github.com/Venture-Miner/wagademy/assets/111710515/db01a1c4-ea75-4089-b395-828ea824fcde)